### PR TITLE
tests: remove java version specific samples testing

### PR DESCRIPTION
**Pull request by anguillanneuf**
This should fix the auth error. The testing project was changed by #298 

Waiting on internal cl/453295389 to be submitted first.

### CODE REVIEW

**Comment by suztomo** [comment]
Kokoro - Test: Samples — Build failed

**Comment by anguillanneuf** [comment]
@suztomo Tests are passing now. Appreciate it if you could stamp.

File: owlbot.py [hunk 1]
```diff
@@ -16,4 +16,11 @@
 
 import synthtool.languages.java as java
 
-java.common_templates()
+java.common_templates(
+    java.common_templates(
+        excludes=[
+            ".kokoro/build.sh",
+            ".github/sync-repo-settings.yaml",
```
Comment (suztomo) [inline owlbot.py:23]: The bot warns about 4 files.

> Warning: This pull request is touching the following templated files:
> 
> .github/sync-repo-settings.yaml
> .kokoro/nightly/samples.cfg
> .kokoro/presubmit/samples.cfg
> samples/snapshot/pom.xml

(If the list is not comprehensive, then we just will get a change later)

**Review by suztomo — APPROVED** [review]

File: owlbot.py (hunk 1 above)
Comment (anguillanneuf) [reply owlbot.py:23]: Uh, I should have excluded two more files in there.

**Pull request merged**